package.json

```diff
@@ -54,7 +54,7 @@
   "dependencies": {
     "@parse/node-apn": "5.1.3",
     "node-adm": "0.9.1",
-    "node-gcm": "github:openphone/node-gcm-axios#master",
+    "node-gcm": "github:openphone/node-gcm-axios#722c343",
```

Comment:
Note that you can use a tag as the version as well.

Reply:
I've been lazy and not creating tags 🙈